src/settings-list.js

```diff
@@ -171,8 +171,8 @@ module.exports = [
         toggle: function(value) {
             if (value === true) {
                 $('body').on('dblclick', '.chat-line', function() {
-                    chat.helpers.translate($(this).find('.message'), $(this).data('sender'), $(this).find('.message').data('raw'));
-                    $(this).find('.message').text('Translating..');
+                    chat.helpers.translate($(this).find('.message'), $(this).data('sender'), $(this).find('.message').text());
```

Comment:
Since we do have a click to restore feature, I'm not sure how we'd handle this then. It might be best to just keep the raw for now. Know of a better way?